capstone/src/main/webapp/style.css

```diff
@@ -53,3 +53,41 @@ button {
   height: 5%;
   margin-left: 50%;
 }
+
+.full-screen-panel {
+  width: 90%;
+  height: 50%;
+  background-color: #c2c2c2;
```

Comment:
nit: change if this can be converted to 3 character hexadecimal notation
https://google.github.io/styleguide/htmlcssguide.html#Hexadecimal_Notation